thonny/plugins/pip_gui.py

```diff
@@ -523,7 +526,49 @@ def write_att(caption, value, value_tag=None):
         if info.get("requires_dist", None):
             # Available only when release is created by a binary wheel
             # https://github.com/pypa/pypi-legacy/issues/622#issuecomment-305829257
-            write_att(tr("Requires"), ", ".join(info["requires_dist"]))
+            requires_dist = info["requires_dist"]
+            assert isinstance(requires_dist, list)
```

Comment:
@aivarannamaa please let me know if you do not want to enforce this kind of type safety at runtime.

Reply:
It's OK.